### ISSUE TITLE
obs-2.6: Repositories tab in UI does not show additional repositories

### ISSUE THREAD

**Issue by tuminoid** [openSUSE/open-build-service#1177]
1. Create project
2. Add repository, say CentOS:6 to that project
3. Edit repo and click "Add additional path to this repository" and add second repository
4. Check from meta that path contains two repositories
```
  <repository name="CentOS_6">
    <path project="openSUSE.org:CentOS:CentOS:7" repository="standard"/>
    <path project="openSUSE.org:CentOS:CentOS-6" repository="standard"/>
...
```
5. Check the repository tab, edit repo -> "Additional package repositories (searched in descending order):" is always empty, no repositories listed

Tested with 2.6.4 appliance. Works with whatever code build.opensuse.org is running though.

**Comment by m4z** [comment]
Might be related to #923.

**Comment by tuminoid** [comment]
@m4z Yes, looks similar. Patch in #923 looks a bit featureish though and not just a bug fix, but it'd be nice to get it merged for 2.6 too. It is a regression after all, like tagged by @mdeniz .

**Comment by m4z** [comment]
Sorry, I was a bit short on time yesterday. I didn't mean to imply in any way that this issue was invalid, I just wanted to add the pointer.

**Comment by tuminoid** [comment]
@m4z Yup, didn't mean to imply so :) Thanks for picking it up, missed it myself. Just took a look at that issue and reported similarities and differences.

**Comment by ChrisBr** [comment]
Fixed for 2.6.9 in #1611

**Comment by tuminoid** [comment]
Thanks @ChrisBr. Seems to be working.
